src/components/member/BusinessDetails/BusinessDetails.jsx

```diff
@@ -239,21 +230,54 @@ const BusinessDetailsList = (props) => {
             if (id) {
               return (
                 <Container key={id} className="business-details-list__container">
-                  <BusinessDetailsPicture
-                    id={id}
-                    imageUrl={imageUrl}
-                    onClickBusinessProfileImage={onClickBusinessProfileImage}
-                    businessCategory={businessCategory}
-                  />
-                  <BusinessDetailsContent
-                    name={name}
-                    description={description}
-                    phone={phone}
-                    products={products}
-                    registeredAddress={registeredAddress}
-                    services={services}
-                    website={website}
-                  />
+                  <div className="col-md-4 col-sm-6 business-box-container business-box-01-container">
+                    <div className="business-circle">
+                      <a onClick={onClickBusinessProfileImage}><IconCamera /></a>
+                    </div>
+                    <div className="business-box-01">
+                      <BusinessDetailsPicture
+                        imageUrl={imageUrl}
+                      />
+                      <div className="box-content">
+                        <BusinessPrimaryDetails
+                          name={name}
+                          description={description}
+                          registeredAddress={registeredAddress}
+                        />
+                      </div>
+                    </div>
+                    <div className="hide-card-details">
+                      <div className="box-content">
+                        <BusinessPrimaryDetails
+                          name={name}
+                          description={description}
+                          registeredAddress={registeredAddress}
+                        />
+                        <div className="general-info">
+                          <div className="heading">
+                            <ul>
+                              <li>
+                                <IconMail />
+                                <div><a href={website} target="_blank">{website}</a></div>

```

Comment:
Render MemberEmail component below
```suggestion
                               <MemberEmail email={email} />
```

Reply:
Done